data/data/bootstrap/vsphere/files/etc/dhcp/dhclient.conf.template

```diff
@@ -0,0 +1,11 @@
+{{if .PlatformData.VSphere.UserProvidedVIPs}}
```

Comment:
i think we should make sure `data/data/bootstrap/vsphere` should have OWNERS file setup.